src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -365,8 +375,52 @@ public boolean isInGoal(int x, int y, int z) {
             logDirect("okay");
             return true;
         }
-        if (msg.equals("farm")) {
-            baritone.getFarmProcess().farm();
+        if (msg.startsWith("farm")) {
+            String rest = msg.substring(4).trim();
+            String[] params = rest.split(" ");
+            int[] paramsInt = new int[params.length];
+            try {
+                for (int i = 0; i < paramsInt.length; i++) {
+                    paramsInt[i] = Integer.parseInt(params[i]);
+                }
+            } catch (NumberFormatException | ArrayIndexOutOfBoundsException | NullPointerException ex) {
```

Comment:
how could arrayindexoutofbounsdexception ever be caused...? or nullpointerexception?